code/modules/jobs/job_types/engineering.dm

```diff
@@ -14,6 +14,9 @@ Chief Engineer
 	selection_color = "#ffeeaa"
 	req_admin_notify = 1
 	minimal_player_age = 7
+	exp_requirements = 600

```

Comment:
These should be config based and default to 0 for downstreams who mirror our PRs